ddtrace/span.py

```diff
@@ -223,10 +223,8 @@ def set_traceback(self, limit=20):
         if (exc_type and exc_val and exc_tb):
             self.set_exc_info(exc_type, exc_val, exc_tb)
         else:
-            tb = ''.join(traceback.format_stack(limit=limit))
-            self.set_tag(PYTHON_STACK, tb)
-            # FIXME[gabin] Delete this line when frontend will display "python.stack" tag correctly
-            self.set_tag(errors.ERROR_STACK, tb)
+            tb = ''.join(traceback.format_stack(limit=limit + 2)[:-2])
```

Comment:
We should also test the last lines to contain the expected piece of code to be sure we stopped the traceback at the right level (maybe one day a revamp of this piece will add a new frame which would shift the result).